### ISSUE TITLE
[Slider] A slider with discrete values cannot be adjusted with swipes in VoiceOver

### ISSUE THREAD

**Issue by jmdetloff** [material-components/material-components-ios#4288]
Double tapping + hold + pan works to adjust this type of slider, but double tap to select then swipe up or down does not.

**Comment by romoore** [comment]
Duplicate of #3040
